diff -burN orig.binutils-2.17/config.sub binutils-2.17/config.sub
--- orig.binutils-2.17/config.sub	2006-01-16 13:34:37.000000000 -0400
+++ binutils-2.17/config.sub	2007-06-17 21:15:17.000000000 -0300
@@ -873,6 +873,10 @@
 	ppc64le-* | powerpc64little-*)
 		basic_machine=powerpc64le-`echo $basic_machine | sed 's/^[^-]*-//'`
 		;;
+	ppu)
+		basic_machine=powerpc64-unknown
+		os=-linux-gnu
+		;;
 	ps2)
 		basic_machine=i386-ibm
 		;;
