diff -burN orig.gcc-4.3-20070608/config.sub gcc-4.3-20070608/config.sub
--- orig.gcc-4.3-20070608/config.sub	2007-06-06 20:23:20.000000000 -0300
+++ gcc-4.3-20070608/config.sub	2007-06-13 19:22:37.000000000 -0300
@@ -877,6 +877,10 @@
 	ppc64le-* | powerpc64little-*)
 		basic_machine=powerpc64le-`echo $basic_machine | sed 's/^[^-]*-//'`
 		;;
+	ppu)
+		basic_machine=powerpc64-unknown
+		os=-linux-gnu
+		;;
 	ps2)
 		basic_machine=i386-ibm
 		;;
